### ISSUE TITLE
asadm-deps/UNKNOWN: No such file or directory

### ISSUE THREAD

**Issue by simonhf** [aerospike/aerospike-admin#35]
I tried to install asadm but got this error. Any ideas?

```
$ git clone https://github.com/aerospike/aerospike-admin.git
$ cd aerospike-admin/
$ sudo ./asadm-deps/install.sh 
./asadm-deps/install.sh: line 50: cd: /home/ec2-user/simon/as/aerospike-admin/asadm-deps/UNKNOWN: No such file or directory
```

**Comment by simonhf** [comment]
In case it helps, these were the steps I used to build and run Aerospike on Amazon Linux:

```
https://github.com/aerospike/aerospike-server

$ git clone https://github.com/aerospike/aerospike-server.git
$ cd aerospike-server/
$ git submodule update --init
$ sudo yum install openssl openssl-devel openssl-static
$ sudo yum install lua lua-devel lua-static
$ sudo yum install python
$ make -j64

$ make init
$ make start

https://github.com/aerospike/aerospike-client-c

$ git clone https://github.com/aerospike/aerospike-client-c.git
$ cd aerospike-client-c/
$ sudo yum install openssl-devel glibc-devel autoconf automake libtool
$ sudo yum install lua-devel
$ sudo yum install gcc-c++ graphviz rpm-build
$ ./install_libev
$ git submodule update --init
$ make EVENT_LIB=libev
$ LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib make EVENT_LIB=libev AS_HOST=127.0.0.1 test
...
SUMMARY
...
185 tests: 185 passed, 0 failed

http://www.aerospike.com/docs/client/c/benchmarks

$ cd benchmarks/
$ make EVENT_LIB=libev
$ LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib make run
$ LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib ./target/benchmarks -h 127.0.0.1 -p 3000 -n test -s demoset -k 1000000 -o S:1400 -w RU,80 -z 4 -L 4,3
hosts:          127.0.0.1
port:           3000
user:           (null)
namespace:      test
set:            demoset
startKey:       1
keys/records:   1000000
bins:           1
object spec:    UTF8 string[1400]
random values:  false
workload:       read 80% write 20%
stop after:     0 transactions
threads:        4
max throughput: unlimited
read timeout:   0 ms
write timeout:  0 ms
max retries:    1
debug:          false
latency:        4 columns, shift exponent 3
shared memory:  false
read replica:   master
read consistency level: one
write commit level: all
Conn pools per node: 1
asynchronous mode:  off
2017-07-11 18:19:30 INFO Add node BB948E04A701B06 127.0.0.1:3000
2017-07-11 18:19:30 INFO Read/write using 1000000 records
2017-07-11 18:19:30 INFO Start 4 generator threads
2017-07-11 18:19:31 INFO write(tps=23696 timeouts=0 errors=0) read(tps=95020 timeouts=0 errors=0) total(tps=118716 timeouts=0 errors=0)
      <=1ms >1ms >8ms >64ms
write  100%   0%   0%    0%
read   100%   0%   0%    0%
2017-07-11 18:19:32 INFO write(tps=23895 timeouts=0 errors=0) read(tps=95110 timeouts=0 errors=0) total(tps=119005 timeouts=0 errors=0)
      <=1ms >1ms >8ms >64ms
write  100%   0%   0%    0%
read   100%   0%   0%    0%
2017-07-11 18:19:33 INFO write(tps=24572 timeouts=0 errors=0) read(tps=98757 timeouts=0 errors=0) total(tps=123329 timeouts=0 errors=0)
      <=1ms >1ms >8ms >64ms
write  100%   0%   0%    0%
read   100%   0%   0%    0%
^C

https://github.com/aerospike/aerospike-admin

$ git clone https://github.com/aerospike/aerospike-admin.git
$ cd aerospike-admin/
$ sudo yum install python27-ply python27-pexpect python27-pyOpenSSL
$ sudo ./asadm-deps/install.sh 
./asadm-deps/install.sh: line 50: cd: /home/ec2-user/simon/as/aerospike-admin/asadm-deps/UNKNOWN: No such file or directory

```

**Comment by hbpatre** [comment]
This is issue of failure to get OS distribution details on Amazon Linux. We will fix this issue in next maintenance cycle.

Following are temporary solutions, 
1) Please install aerospike-tools package from [Aerospike Download page](http://www.aerospike.com/download/tools/3.13.0.1/). 
2) Please run install.sh files from all subfolders of ./asadm-deps/rpm with sudo permission.

**Comment by hbpatre** [comment]
This issue is fixed and released asadm-0.1.12.
